src/validations/request-approval-rejection.validation.js

```diff
@@ -0,0 +1,40 @@
+import joiBase from '@hapi/joi';
```

Comment:
Already gave feedback that files should not be created in isolation. They should be seen as a group/collection.

Rename this file appropriately.